docs/reference/how-to/disk-usage.asciidoc

```diff
@@ -136,20 +136,42 @@ PUT index
 // CONSOLE
 
 [float]
+=== Watch your shard size
+
+Larger shards are going to be more efficient at storing data. To increase the size of your shards, you can decrease the number of primary shards in an index by <<indices-create-index,creating indices>> with less primary shards, creating less indices (e.g. by leveraging the <<indices-rollover-index,Rollover API>>), or modifying an existing index using the <<indices-shrink-index,Shrink API>>.
+
+Keep in mind that very large shard sizes come with drawbacks, such as longer recovery times.
+
+[float]
 === Disable `_all`
 
 The <<mapping-all-field,`_all`>> field indexes the value of all fields of a
 document and can use significant space. If you never need to search against all
 fields at the same time, it can be disabled.
 
 [float]
+=== Disable `_source`
+
+The <<mapping-source-field,`_source`>> field stores the original JSON body of the document. If you don’t need access to it you can disable it. However, APIs that needs access to `_source` such as update and reindex won’t work.
+
+[float]
 === Use `best_compression`
 
 The `_source` and stored fields can easily take a non negligible amount of disk
 space. They can be compressed more aggressively by using the `best_compression`
 <<index-codec,codec>>.
 
 [float]
+=== Force Merge
+
+Elasticsearch stores data in segments. Segments make up a Lucene index - a shard in Elasticsearch. The <<indices-forcemerge,`_forcemerge` API>> can be used to reduce the number of segments per shard. In many cases, the number of segments can be reduced to one per shard by setting `max_num_segments=1`.
```

Comment:
Perhaps turn this around a bit: Elasticsearch stores data in shards. Shards are Lucene indices and are composed of segments. Segments are the actual files on disk, etc.